roles/cloud-bootstrap/tasks/accounts.yml

```diff
@@ -0,0 +1,76 @@
+---
+- name: Ensure projects
+  os_project:
+    cloud: "{{ cloud }}"
+    name: "{{ item }}"
+  with_items:
+    - "{{ prod_project }}"
+    - "{{ nodepool_project }}"
+    - "{{ users.username }}"
```

Comment:
This will probably fail. `users.username` has no meaning at this point. I think you may want to do some concatenation for the three variable sets, or since that's awkward, just do one task that has the `prod_project` and `nodepool_project` items, and a second task that loops over `users` and uses `item.username`. No need to be really clever, just make it clear to understand.